### PR TITLE
VACMS-1239: Removing field_office from 4 node types.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #1239 
This is step 4 of process = remove field_office from Events, News releases, Publications and Stories.
**Step 3 (the front end consumption switch over) was merged into vets-website yesterday. Should be fine to merge in now, but to be safe, we should wait until this afternoon to merge this in**

## Testing done
Dst

## QA steps
As an admin, visit the following paths, and visually verify that field_office does not appear:
 - [x] `/admin/structure/types/manage/event/fields`
 - [x] `/admin/structure/types/manage/news_story/fields`
 - [x] `/admin/structure/types/manage/press_release/fields`
 - [x] `/admin/structure/types/manage/outreach_asset/fields`

### CODE REVIEW

**Comment by swirtSJW** [comment]
Looks like there is a metalsmith error 
image

**Comment by ethanteague** [comment]
> Looks like there is a metalsmith error
> image

My bad, @swirtSJW - there are unused entries for field_office that are breaking the build (in other templates). This pr removes: https://github.com/department-of-veterans-affairs/vets-website/pull/12047

**Comment by swirtSJW** [comment]
This one has to wait on this to clear first on the front end https://github.com/department-of-veterans-affairs/vets-website/pull/12047

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install [hunk 1]
```diff
@@ -267,3 +268,21 @@ function va_gov_backend_update_8005() {
...
+    'news_story',
+    'press_release',
+    'publication',
+  ];
+  foreach ($bundles as $bundle) {
+    $field = FieldConfig::loadByName('node', $bundle, 'field_office');
+    if ($field) {
+      $field->delete();
+    }
+  }
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.install:287]: Minor:  No action required:  Update hooks should not run silently.  They should log the action of what was done.

**Pull request merged**